tests/sentry/tasks/integrations/github/test_pr_comment.py

```diff
@@ -157,43 +159,22 @@ def test_multiple_prs(self):
         groupowner_1 = self.add_groupowner_to_commit(commit_1, self.project, self.user)
         groupowner_2 = self.add_groupowner_to_commit(commit_2, self.project, self.user)
 
-        results = pr_comment.pr_to_issue_query()
-
+        results = pr_comment.pr_to_issue_query(pr_1.id)
         assert results[0] == (
             self.gh_repo.id,
             pr_1.key,
             self.organization.id,
             [groupowner_1.group_id],
         )
-        assert results[1] == (
+
+        results = pr_comment.pr_to_issue_query(pr_2.id)
+        assert results[0] == (
             self.gh_repo.id,
             pr_2.key,
             self.organization.id,
             [groupowner_2.group_id],
         )
 
-    def test_non_gh_repo(self):
-        """Repos that aren't GH should be omitted"""
-
-        commit = self.add_commit_to_repo(self.not_gh_repo, self.user, self.project)
-        self.add_pr_to_commit(commit)
-        self.add_groupowner_to_commit(commit, self.project, self.user)
-
-        results = pr_comment.pr_to_issue_query()
-
-        assert len(results) == 0
-
-    def test_pr_too_old(self):
```

Comment:
removed these tests because we will be filtering these PRs out before queuing the task